lib/ansible/modules/cloud/amazon/kinesis_stream.py

```diff
@@ -695,14 +700,61 @@ def retention_action(client, stream_name, retention_period=24,
     return success, err_msg
 
 
-def update(client, current_stream, stream_name, retention_period=None,
+def update_shard_count(client, stream_name, number_of_shards=1, check_mode=False):
```

Comment:
This is updating the number of shards correctly and maintains the style of the rest of this module. Thanks for working on this.